ml4gw/transforms/decimator.py

```diff
@@ -0,0 +1,137 @@
+import torch
+
+
+class Decimator(torch.nn.Module):
+    """
+    Downsample/decimate timeseries according to a schedule.
+
+    The schedule is a tensor of shape (N, 3), where each row is:
+    [start_time, end_time, target_sample_rate]
+
```

Comment:
add in the docstring that this is a naive decimator that is not as sophisticated in the sense it does not use any IIR/FIR filters.